ios/RNOneginiSdk.swift

```diff
@@ -221,7 +222,7 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
                                     scopes: [String],
                                     resolver resolve: @escaping RCTPromiseResolveBlock,
                                     rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
-        let profile: ONGUserProfile = ONGClient.sharedInstance().userClient.userProfiles().first(where: { $0.value(forKey: "profileId") as! NSObject == profileId })!;
+        let profile = ONGClient.sharedInstance().userClient.userProfiles().first(where: { $0.value(forKey: "profileId") as! NSObject == profileId })!;
```

Comment:
Can you define an object property (`let userClient = ONGClient.sharedInstance().userClient`) and use it everywhere? Or even function that would return `profile` for provided `profileId`?

Reply:
I could but this is not a scope of the story.

Reply:
I've refactored nevertheless 🤟